samples/bot-proactive-messaging-teamsfx/bot/teamsBot.js

```diff
@@ -32,7 +38,7 @@ class TeamsBot extends TeamsActivityHandler {
       this.addConversationReference(context.activity);
 
       // Echo back what the user said
-      await context.sendActivity(`You sent '${context.activity.text}'. Navigate to http://{your-domain}/api/notify to proactively message everyone who has previously messaged this bot.`);
+      await context.sendActivity(`You sent '${context.activity.text}'. Navigate to ${process.env.PROVISIONOUTPUT__BOTOUTPUT__SITEENDPOINT}/api/notify to proactively message everyone who has previously messaged this bot.`);
```

Comment:
Will it work without provisioning?
Support I am not running the app using toolkit and trying it locally without provisioning (Normal app running using Ngrok/dev tunnel)?

Reply:
Please test it locally without toolkit once.